exercise-app/pom.xml

```diff
@@ -23,5 +23,11 @@
       <artifactId>exercise-core</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
Since this is used also in the other project (-core) it is worthwhile to move this dependency in the parent pom